.gitpod.dockerfile

```diff
@@ -2,8 +2,13 @@ FROM gitpod/workspace-postgres
 
 # Install Ruby
 ENV RUBY_VERSION=2.7.1
+RUN rm /home/gitpod/.rvmrc && touch /home/gitpod/.rvmrc && echo "rvm_gems_path=/home/gitpod/.rvm" > /home/gitpod/.rvmrc
 RUN bash -lc "rvm install ruby-$RUBY_VERSION && rvm use ruby-$RUBY_VERSION --default"
 
+# Install Node
```

Comment:
Hm, I'm not sure if this is necessary. The image we use (`gitpod/workspace-postgres`) inherits from `gitpod/workspace-full` which includes Node. On my still working cached pod of master the following Node version is available:

```
gitpod /workspace/dev.to $ node
Welcome to Node.js v12.16.1.
```

Reply:
weird,  I was getting this error.  Yea I should probably check which node versions are installed with nvm prior and see if I can just reassign to v12.16 instead of reinstalling

<img width="902" alt="2020-06-09 00 35 51_ALrRAp png" src="https://user-images.githubusercontent.com/12533693/84155216-b5e2a080-aa2d-11ea-9c50-8d960d30680e.png">

Reply:
https://hub.docker.com/r/gitpod/workspace-full/dockerfile this says it uses `v8.14.0`? so confused

Reply:
then, clearing up the node version error, get this last error :(
<img width="852" alt="2020-06-09 08 53 26_94wvM5 png" src="https://user-images.githubusercontent.com/12533693/84156127-bfb8d380-aa2e-11ea-8e89-2d35b5d04be1.png">

Reply:
feel free to commit to this branch if you have ideas as well

Reply:
oh wait 🤦 it was still using 12.18.3 I didn't change the path or .nvmrc to ensure it uses 12.16.3. that fixed the above

Reply:
> weird, I was getting this error. Yea I should probably check which node versions are installed with nvm prior and see if I can just reassign to v12.16 instead of reinstalling
> 
> <img alt="2020-06-09 00 35 51_ALrRAp png" width="902" src="https://user-images.githubusercontent.com/12533693/84155216-b5e2a080-aa2d-11ea-9c50-8d960d30680e.png">
> 
> I think its a caching thing again, that base img must have changed their node version?

You can fix this, by changing the version number of the node to 12.18.x in the package.json